lib/qesapdeployment.pm

```diff
@@ -578,7 +578,7 @@ sub qesap_execute {
 
 =item B<RETRIES> - number of retries in case of expected error
 
-=item B<ERROR_STRING> - error string to look for
+=item B<ERROR_LIST> - list of error strings to look for
```

Comment:
Or maybe please expand this one. 
```suggestion
=item B<ERROR_LIST> - list of error strings to look for (where). In case (one|all) match function is doing (what).
```